skribenten-web/frontend/src/Brevredigering/LetterEditor/actions/switchFontType.ts

```diff
@@ -0,0 +1,46 @@
+import { produce } from "immer";
+
+import type { FontType } from "~/types/brevbakerTypes";
+import { handleSwitchContent, handleSwitchTextContent } from "~/utils/brevbakerUtils";
+
+import type { Action } from "../lib/actions";
+import type { LetterEditorState } from "../model/state";
+import { isItemContentIndex } from "../model/utils";
+import type { LiteralIndex } from "./model";
+
+export const switchFontType: Action<LetterEditorState, [literalIndex: LiteralIndex, fontType: FontType]> = produce(
```

Comment:
Så det blir kun mulig å endre formattering for hele blokken eller item?

Reply:
Per nå er det implementert at det er hele blokken. Når jeg tenker over det, så er det kanskje litt idiotisk?